### PR TITLE
Story/ep 1790

### PR DESCRIPTION
**Pull request by dan-qc**

### CODE REVIEW

File: packages/earth-admin/src/components/org-switcher/OrgSwitcher.tsx [hunk 1]
```diff
@@ -62,7 +62,7 @@ const OrgSwitcher = (props) => {
           </a>
         </li>
         {!!permissions &&
-          Object.keys(permissions).map((g, i) => (
+          Object.keys(permissions).filter(k => k !== '*').map((g, i) => (
```
Comment (ancashoria) [inline packages/earth-admin/src/components/org-switcher/OrgSwitcher.tsx]: would be useful to name the filter function. Could it be `notOwner` or `withoutOwner` or whatever makes the code more readable

File: packages/earth-admin/src/components/organizations/organization-edit/OrganizationEdit.tsx [hunk 2]
```diff
@@ -65,27 +77,30 @@ export default function OrganizationEdit(props: OrganizationEditProps) {
                 Organization name
               </label>
               <input
-                ref={register}
+                ref={register({
+                  required: true,
```
Comment (ancashoria) [inline packages/earth-admin/src/components/organizations/organization-edit/OrganizationEdit.tsx:81]: Does this required flag affect the edit organization flow? Or is it no longer required if the field is diabled?

File: packages/earth-map/src/components/header/component.tsx [hunk 3]
```diff
@@ -143,7 +143,7 @@ const Header = (props: IProps) => {
         </li>
         <li className="ng-form ng-form-dark">
           <div className="ng-padding-medium-horizontal ng-padding-top">
-            {Object.keys(roles).map((g, i) => (
+            {Object.keys(roles).filter(k => k !== '*').map((g, i) => (
```
Comment (ancashoria) [inline packages/earth-map/src/components/header/component.tsx]: same as above

File: packages/earth-admin/src/components/organizations/organization-edit/OrganizationEdit.tsx (hunk 2 above)
Comment (dan-qc) [reply packages/earth-admin/src/components/organizations/organization-edit/OrganizationEdit.tsx:81]: In "edit mode", the value will be there already - it's always required, to make it more clear.

File: packages/earth-admin/src/components/org-switcher/OrgSwitcher.tsx [hunk 4]
```diff
@@ -40,6 +40,16 @@ const OrgSwitcher = (props) => {
...
+  const getAvailableOrgs = () => {
+    const specialPermissions = [
+      '*' // super-admin
+    ];
+
+    return Object
+      .keys(permissions)
+      .filter(permission => !specialPermissions.includes(permission));
+  };
+
```
Comment (ancashoria) [inline packages/earth-admin/src/components/org-switcher/OrgSwitcher.tsx]: - Reuse this function by moving it into utils
- pass permissions as parameter
- add types for parameter and return value 
- add a description comment above the function

**Review by ancashoria — APPROVED** [review]

**Pull request merged**